src/test/docker/noble/Dockerfile

```diff
@@ -39,14 +39,17 @@ RUN apt-get update \
         make \
         cmake \
         ninja-build \
+        gfortran \
         clang-18 \
         clang-tools-18 \
 # default version
         gcc-13 \
         g++-13 \
+        gfortran-13 \
 # newest
         gcc-14 \
         g++-14 \
+        gfortran-14 \
```

Comment:
Removing libmpich-dev makes sense!

> Also, add installs of gfortran to prevent configure errors when building MPICH from source.

Weren't we always buliding mpich from source in this image?  How was that working without fortran?
We could configure mpich with `--disable-fortran` in the build script rather than pull in (two versions of?)  the fortran tool chain.

Reply:
We could definitely just disable Fortran if that's preferred. I have no idea how it was working before. I just know that, when I built the image myself before making any changes, the build failed because it couldn't find Fortran.

Reply:
Just pushed a commit that disables Fortran

Reply:
> Weren't we always buliding mpich from source in this image? How was that working without fortran?
We could configure mpich with --disable-fortran in the build script rather than pull in (two versions of?) the fortran tool chain.

One worry is that someone will come along and want to build or run a test that uses the MPICH fortran and we'll get another issue report that it doesn't work. I've already built the `fluxrm/testenv:noble` image with the gfortran included and pushed that to docker hub, so @ilumsden I wonder if you could try your experiments against the latest `noble` image?

We shouldn't be too worried about the size of the noble image here since it already pulls in _a lot_ of other stuff...

Reply:
Great point.  Sorry for that diversion!  Back to the original OK @ilumsden ?